velox/functions/prestosql/aggregates/tests/MinMaxByAggregationTest.cpp

```diff
@@ -309,8 +299,13 @@ std::string asSql(bool value) {
 void MinMaxByAggregationTestBase::SetUp() {
   AggregationTestBase::SetUp();
   AggregationTestBase::disallowInputShuffle();
-
-  for (const TypeKind type : kSupportedTypes) {
+  std::vector<TypePtr> supportedTypes;
+  for (const auto kind : kSupportedTypes) {
+    supportedTypes.push_back(fromKindToScalerType(kind));
```

Comment:
Should we generate the column names here to align with the kSupportedTypes size?